exercises/practice/reverse-string/stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
```

Comment:
This file should not be committed.

I have proposed to have these files be automatically ignored in #1133.